### ISSUE TITLE
SEV-ES support missing for kernel version 5.10?

### ISSUE THREAD

**Issue by alina-weber** [AMDESE/AMDSEV#56]
I installed the latest versions for kernel/linux (sev-es-v3), qemu (sev-es-v12) and ovmf (sev-es-v27). When booting the 5.10 kernel I get the following dmesg output regarding SEV*: 
![Screenshot from 2020-12-01 13-04-49](https://user-images.githubusercontent.com/44259717/100739068-d553f800-33d6-11eb-9c78-7c65b5b85d42.png)
The message from svm stating the available ASID's is missing for both SEV and SEV-ES. I can start a VM with SEV enable but when triyng to launch a VM with SEV-ES I get the following output:
![Screenshot from 2020-12-01 13-14-02](https://user-images.githubusercontent.com/44259717/100739381-485d6e80-33d7-11eb-900d-4f34941fc681.png)

I also noticed that in svm.c the implementation of `sev_launch_update_vmsa()` is missing. As VMSA is related to SEV-ES, is it possible that kernel 5.10 does not support SEV-ES yet?

**Comment by tlendacky** [comment]
The upstream 5.10 Linux tree has only SEV-ES guest support. That means 5.10 can run as an SEV-ES guest but it cannot launch an SEV-ES guest when run as the hypervisor. The hypervisor support is in the sev-es-v3 branch of the Linux tree you cloned from github. You will need to boot the sev-es-v3 kernel as the hypervisor and then you can boot a guest with either the sev-es-v3 kernel or a 5.10 kernel.

**Comment by alina-weber** [comment]
I guess my description was imprecise. I booted the sev-es-v3 kernel. The built kernel of the sev-es-v3 branch is also an 5.10 kernel. VMs with kernel versions lower than 5.10 crashed after boot-up, even though they worked with an older kernel from the sev-es branch (e.g. 5.6). I also updated my VM to the 5.10 kernel from kernel.org and now I can launch the VM running the 5.10 kernel from kernel.org with the hypervisor running the sev-es-v3 5.10 kernel. Thanks for your help!

**Comment by alina-weber** [comment]
I guess my description was imprecise: I booted the sev-es-v3 kernel. The built kernel of the sev-es-v3 branch is also an 5.10 kernel. VMs with kernel versions lower than 5.10 crashed after boot-up, even though they worked with an older kernel from the sev-es branch (e.g. 5.6). I also updated my VM to the 5.10 kernel from kernel.org and now I can launch the VM running the 5.10 kernel from kernel.org with the hypervisor running the sev-es-v3 5.10 kernel. Thanks for your help!
